docs/source/reference-material/hooks-api.rst

```diff
@@ -1,6 +1,6 @@
-================
-Life Cycle Hooks
-================
+=========
+Hooks API
```

Comment:
The `LAZY INITIAL STATE` section in here is important enough to move up to the `Managing State` tab whenever that gets developed.